golangci-lint/main.go

```diff
@@ -0,0 +1,9 @@
+package main
+
+import (
+	"strconv"
+)
+
+func main() {
+	strconv.Atoi("255")
```

Comment:
🚫 **[golangci-lint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Error return value of `strconv.Atoi` is not checked (errcheck)